_layouts/production.html

```diff
@@ -1,35 +1,89 @@
 ---
 layout: default
 ---
-  <div class="production-posts">
-      <div class="production-post spacing">
-        <h1>{{ page.title }}</h1>
-          <b><span class="date">
-            {{ page.dates_string }} | {{page.location}}
-          </span></b>
-        </p>
-        {{ page.synopsis }}
-        <h3>Production Team:</h3>
-        <ul>
-        {% for production in page.production %}
+
+<div class="container">
+  <div id="main_area">
+    <!-- Slider -->
+    <div class="row">
+      <div class="col-xs-12" id="slider">
+        <!-- Top part of the slider -->
+        <div class="row">
+          <div class="col-sm-8" id="carousel-bounding-box">
+            <div class="carousel slide" id="myCarousel">
+              <!-- Carousel items -->
+              <div class="carousel-inner">
+                {% for image in page.images %}
+                {% if forloop.index == 1 %}
+                <div class="active item" data-slide-number="{{forloop.index }}">
+                  {% else %}
+                  <div class="item" data-slide-number="{{forloop.index }}">
```

Comment:
These lines shouldn't be nested, since they're in an if statement. Ex:
```
{% for image in page.images %}
    {% if forloop.index == 1 %}
         <div class="active item" data-slide-number="{{forloop.index }}">
    {% else %}
        <div class="item" data-slide-number="{{forloop.index }}">
    {% endif %}
    <img src="{{site.url}}/{{image.url}}"></div>
{% endfor %}
```